src/server/server/frontendhelpers.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
```

Comment:
The name "frontendhelpers" does not seems very descriptive and a catchall name.

Since there is for now only password related thing, we can call the module (file) "passwordHelpers" and make the two methods standalone (without class) ?
And the prefix "frontend" doesn't seem to add value because it's already on frontend ("server") module.

By the way and not related to this commit (I'll add an issue if you agree) I think we should rename "frontendstruct" to "sharedstruct" which is what we use those structs for : to share structs between several modules